### PR TITLE
Create add rooms e2e test

### PR DESCRIPTION
**Pull request by GuilhermBrSp**
## What ❓ 

This PR adds e2e test for add rooms use case, including a lot of changes on how we start applications on e2e (now using containers) and configure clients for tests.

### CODE REVIEW

File: e2e/framework/maestro/exec/exec.go [hunk 1]
```diff
@@ -84,3 +84,25 @@ func ExecGoRun(dir string, env []string, externalArgs ...string) (*Cmd, error) {
 
 	return c, nil
 }
+
+func ExecRun(dir string, command string, args ...string) (*Cmd, error) {
```
Comment (luizmiranda7) [inline e2e/framework/maestro/exec/exec.go:88]: I would rename it to something like: ExecSysCmd.

Comment (luizmiranda7) [reply e2e/framework/maestro/exec/exec.go:88]: Also, there's a function called ExecGoRun, I would also change it for something like: ExecGoCmd

File: e2e/suites/management/add_rooms_test.go [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+	"github.com/go-redis/redis"
+	roomStorageRedis "github.com/topfreegames/maestro/internal/adapters/room_storage/redis"
+	"github.com/topfreegames/maestro/internal/core/entities/game_room"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/topfreegames/maestro/e2e/framework/maestro"
+
+	"github.com/topfreegames/maestro/e2e/framework"
+	maestrov1 "github.com/topfreegames/maestro/pkg/api/v1"
```
Comment (luizmiranda7) [inline e2e/suites/management/add_rooms_test.go:40]:
```suggestion
	maestroApiV1 "github.com/topfreegames/maestro/pkg/api/v1"
```

maestrov1 is not so clear to me.

File: e2e/suites/management/add_rooms_test.go [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+			}, 5*time.Second, time.Second)
+
+			addRoomsRequest := &maestrov1.AddRoomsRequest{
+				SchedulerName: schedulerName,
+				Amount:        1,
+			}
+			addRoomsResponse := &maestrov1.AddRoomsResponse{}
+			err = apiClient.Do("POST", fmt.Sprintf("/schedulers/%s/add-rooms", schedulerName), addRoomsRequest, addRoomsResponse)
+
+			require.Eventually(t, func() bool {
```
Comment (luizmiranda7) [inline e2e/suites/management/add_rooms_test.go:121]: There're more similar code blocks, WDYT about extract it as a function?

File: e2e/suites/management/add_rooms_test.go [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func TestAddRooms(t *testing.T) {
+	framework.WithClients(t, func(apiClient *framework.APIClient, kubeclient kubernetes.Interface, redisClient *redis.Client, maestro *maestro.MaestroInstance) {
+
+		roomsStorage := roomStorageRedis.NewRedisStateStorage(redisClient)
+
+		t.Run("when created rooms does not reply its state back then it finishes the operation successfully", func(t *testing.T) {
```
Comment (luizmiranda7) [inline e2e/suites/management/add_rooms_test.go:50]: This test description is not so clear to me, reading the test code, I believe that you're just testing the happy path, that's right?

Comment (GuilhermBrSp) [reply e2e/suites/management/add_rooms_test.go:50]: Yes, even if the game rooms does no reply its ready state back or some error occur when each of the rooms, we consider the operation finished with success. The only sad path that I can identify is when the scheduler is not found when adding rooms, I'm going to add this test case!.

**Review by gabrielcorado — APPROVED** [review]

File: e2e/suites/management/add_rooms_test.go (hunk 4 above)
Comment (GuilhermBrSp) [reply e2e/suites/management/add_rooms_test.go:50]: c350ce2

File: e2e/suites/management/add_rooms_test.go (hunk 2 above)
Comment (GuilhermBrSp) [reply e2e/suites/management/add_rooms_test.go]: c350ce2

File: e2e/framework/maestro/exec/exec.go (hunk 1 above)
Comment (GuilhermBrSp) [reply e2e/framework/maestro/exec/exec.go]: c350ce2

**Review by luizmiranda7 — APPROVED** [review]

**Pull request merged**